src/chart_types/partition_chart/state/selectors/tooltip.ts

```diff
@@ -73,7 +76,7 @@ export const getTooltipInfoSelector = createCachedSelector(
           specId: pieSpec.id,
           key: pieSpec.id,
         },
-        value: valueFormatter(valueGetterFun(shape)),
+        value: valueFormatter(primaryValueGetterFun(shape)) + ' (' + percentFormatter(percentValueGetter(shape)) + ')',
```

Comment:
nit: can you use a template string here?
```suggestion
        value: `${valueFormatter(primaryValueGetterFun(shape))} (${percentFormatter(percentValueGetter(shape))})`,
```